model.md

```diff
@@ -1,5 +1,23 @@
 # Model
 
+
+## Principles
+
+- People should be able to possess a copy of the data they rely on.
+- Individuals and communities should be able to control what content is held on their machines and on their networks
+- We must make the web resilient and resistant to monopolies
+- Managing the web's content through collective action is a better approach than relying on monopolies
+
+## Activities
+
+### Decentralizing Data: Taking Possession of the Data you Rely On
+
+### Coordinating to Possess Data Together
+
+Applying a Participatory form of Library Practice
```

Comment:
I feel like there is so much in here that I would want to pull on radical/grassroots archival and library practice. In part I'm especially thinking about terminology.

Reply:
I guess I'm wondering if this can be separated off for now?

Reply:
👍 removed the incomplete stuff. Put it in #42 for follow up.